H_ecto-models.md

```diff
@@ -517,6 +517,86 @@ end
 
 That's the end of our walk-through of Ecto usage in our controller actions. There is quite a bit more that Ecto models can do. Please take a look at the [Ecto documentation](http://hexdocs.pm/ecto/) for the rest of the story.
 
+### Data Relationship and Dependencies
+
+Suppose we are building a very simple video-sharing web application, in addition to having users on our site, we might also want to have videos. We asked Phoenix to scaffold a `Video` model for us:
+
+```console
+$ mix phoenix.gen.model Video videos name:string approved_at:datetime description:text likes:integer views:integer
```

Comment:
What would you think about adding `user:belongs_to` at the end of this `phoenix.gen.model` call? It's just a little added insurance that there won't be any frustrating errors for users later on. That would make the whole call:

`$ mix phoenix.gen.model Video videos name:string approved_at:datetime description:text likes:integer views:integer user:belongs_to`